Tests/TestHelpers.swift

```diff
@@ -171,6 +181,20 @@ extension GitHubUserContent: TargetType {
     }
 }
 
+// MARK: - Upload Multipart Helpers
+
+extension HTTPBin {
+    static func createMultipartFormData() -> [MultipartFormData] {
+        let url = Bundle(for: MoyaProviderSpec.self).url(forResource: "testImage", withExtension: "png")!
```

Comment:
I know it's a helper, but maybe `fatalError` here too?

Reply:
Sounds like a good idea to me 👍